core/state_processor_test.go

```diff
@@ -302,6 +303,90 @@ func TestStateProcessorErrors(t *testing.T) {
 	}
 }
 
+// TestStateProcessorErrors tests the output from the 'core' errors
```

Comment:
nit: TestStateProcessorErrors -> TestBadTxAllowListBlock

The comment also mentions the errors in `core/errors.go` which aren't used here, so we should update that part of the comment too.